docs/developer-guide/syntax-reference.rst

```diff
@@ -1297,7 +1297,7 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | UCASE                  |  ``UCASE(col1)``                                                          | Convert a string to uppercase.                    |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
-| URL_DECODE_PARAM       |  ``URL_DECODE_PARAM(col1)``                                               | Unescapes the URL-param-encoded value in col1.    |
+| URL_DECODE_PARAM       |  ``URL_DECODE_PARAM(col1)``                                               | Unescapes the URL-param-encoded[1]_ value in col1.|
```

Comment:
looks like footnotes don't display on github :( I'll look into another way of updating this

Reply:
No, but you can link to a common URL, e.g.

```markdown
See [URL rfc][1] for more info.

[1] https://tools.ietf.org/html/rfc3986
```